index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Harmattan&family=Inter:wght@100&family=Poppins:wght@200&family=Roboto+Serif:wght@100&family=Roboto:ital,wght@0,400;1,100&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>My-Portfolio</title>
+  </head>
+  <body>
+    <section>
```

Comment:
[REQUIRED] Kindly fix the weird behavior of the page when the user scroll on y axis
![image](https://user-images.githubusercontent.com/74744787/158691834-178ccd91-8e40-4017-975c-0dc8e304ea4d.png)